examples/example-smart-contract/src/main.rs

```diff
@@ -1,17 +1,20 @@
 #![no_std]
 #![no_main]
 
+use parity_scale_codec::{Decode, Encode};
+
 qf_polkavm_sdk::host_functions!();
 
 #[polkavm_derive::polkavm_export]
 extern "C" fn main(op: u32) -> u64 {
```

Comment:
Need to add scale args. As I can see we are using u32.

Reply:
fixed.